packages/scaffolding-cli/templates/src/ssr/interfaces/auth.interface.tsx

```diff
@@ -0,0 +1,20 @@
+export interface UserType {
+    displayName: string
+    id: string
+    user_id: string
+    provider: string
+    picture: string
+    locale: string
+    nickname: string
+    name: name_type
+    email: EmailType[]
+}
+
+export interface name_type {
+    familyName: string
+    givenName: string
+}
+
+export interface EmailType {
+    value: string
+}
```

Comment:
Check your naming conventions here. Camel, snake and capitalised cases.

Reply:
Fixed typo. Capitals always for exported interfaces.